_config.yml

```diff
@@ -26,7 +26,19 @@ collections_dir: content
 collections:
   help:
     output: true
-    # permalink: /:collection/:path/
+    permalink: /:collection/:path/
+  landing:
+    output: true
+    permalink: /:path/
+  developers: 
+    output: true
+    permalink: /:path/
+  partners:
+    output: true
+    permalink: /:path/
+  policy: 
+    output: true
+    permalink: /:path/
   en:
```

Comment:
Do the `en`, `es`, and `fr` collections exist anymore, since we no longer have the `content/_en`, `content/_es`, and `content/_fr` folders?

Reply:
they do not exist anymore, so I can remove this